### ISSUE TITLE
What would it take for apple3 target?

### ISSUE THREAD

**Issue by tschak909** [cc65/cc65#846]
What would it take to alter e.g. the apple2 target to work on apple3? I don't think it would be too much, but I lack some insight into e.g. the bank switching, and differences between SOS and ProDOS.. thought I may possibly start some discussion on the subject.

-Thom

**Comment by polluks** [comment]
At least we have support in ca65 https://github.com/david-schmidt/apple-iii-cffa3000-driver/blob/master/src/CFFA3000.s

**Comment by oliverschmidt** [comment]
@polluks: I must admit that I don't get your point. The Apple /// has a 6502 and therefore you can have ca65 create code for it - but that' true for all 6502 machines. What beyond that do want to prove with the link above?

**Comment by polluks** [comment]
@oliverschmidt Indeed, Thom wants C, not a driver written in assembler. Never the less we can see how to use o65 as an intermediate format.
Isn't David's https://github.com/david-schmidt/apple3rtr a good starting point?

**Comment by oliverschmidt** [comment]
* I can't assess if o65 has benefits for the idea to support the Apple /// with cc65.
* I can't see how https://github.com/david-schmidt/apple3rtr is a starting point for supporting the Apple /// with cc65. It seems to be a starting point to do something with that machine in general.
* I'm personally not motivated to work on supporting the Apple ///.

**Comment by groessler** [comment]
If somebody ships an Apple /// to me I would work on supporting the system. :-)

**Comment by tschak909** [comment]
hmmm I know somebody with one... shipping MIGHT be an issue... ;)

**Comment by groessler** [comment]
> shipping MIGHT be an issue...
Of course I would pay it if I could keep the machine afterwards :-)

**Comment by groessler** [comment]
I would pay the shipping costs of course, If I could keep the machine afterwards...

**Comment by polluks** [comment]
Just for the record http://robjapple.blogspot.com/2018/12/apple-drivers-and-new-way-of-cross.html
